### PR TITLE
refactor(generator migration): migrate repo to latest generator build

### LINKED ISSUE #30: auro-dropdown: develop styles for onDark version
## Is your feature request related to a problem? Please describe.

`auro-dropdown` needs all styling added for `onDark` version.

## Describe the solution you'd like

Styles need to match the [figma document](https://www.figma.com/file/3TQhiKjQqQOKYqaqcO1zj2/Dropdown?node-id=906%3A753).

### LINKED ISSUE #31: auro-dropdown: update to generator v3.11.1
Repo needs to be updated to latest generator version.

### PR DESCRIPTION
**Pull request by jason-capsule42**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #31 

## Summary:

This change updates the repository to base off of generator v3.11.1

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [ ] Revision of an existing capability
- [x] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: .github/settings.yml [hunk 1]
```diff
@@ -3,13 +3,13 @@ repository:
   # See https://github.com/apps/settings for app settings
 
   # A short description of the repository that will show up on GitHub
-  description: An auroLabs custom element w/an experimental API. You've been warned.
+  description: Custom element that ....
```
Comment (blackfalcon) [inline .github/settings.yml:6]: Please update description

File: .github/workflows/testPublish.yml [hunk 2]
```diff
@@ -37,7 +37,7 @@ jobs:
           persist-credentials: false
       - uses: actions/setup-node@v1
         with:
-          node-version: 16
+          node-version: 14
```
Comment (blackfalcon) [inline .github/workflows/testPublish.yml]: This should be `16`?

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-# auro-dropdown
+# Dropdown
 
-`<auro-dropdown>` is an experimental [HTML custom element](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements) only available as a PRE RELEASE. Users of this element understand the risks of using a pre-released version of this element.
+`<auro-dropdown>` is a [HTML custom element](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements) for the purpose of ...
```
Comment (blackfalcon) [inline README.md:3]: Please update to use the description we want to go live with.

File: README.md [hunk 4]
```diff
@@ -56,38 +58,53 @@ Since the legacy bundle includes many polyfills that are not needed by modern br
...
 ```
 
+## auro-dropdown use cases
+
+The `<auro-dropdown>` element should be used in situations where users may:
+
+* ...
+* ...
+* ...
+
```
Comment (blackfalcon) [inline README.md:72]: Please add use cases or delete, up to you.

File: docs/api.md [hunk 5]
```diff
@@ -2,21 +2,17 @@
...
-| Method                        | Type       |
-|-------------------------------|------------|
-| `dispatchEventDropdownToggle` | `(): void` |
-| `fixWidth`                    | `(): void` |
-| `hide`                        | `(): void` |
-| `show`                        | `(): void` |
+| Method | Type       |
+|--------|------------|
+| `hide` | `(): void` |
+| `show` | `(): void` |
```
Comment (blackfalcon) [inline docs/api.md]: These should be `@private` methods.

File: src/auro-dropdown.js [hunk 6]
```diff
@@ -48,9 +48,15 @@ class AuroDropdown extends LitElement {
...
     return {
+      /**
+       * @private
+       */
       placement:     { type: String },
       toggle:        { type: Boolean },
       chevron:       { type: Boolean },
+      /**
+       * @private
+       */
```
Comment (blackfalcon) [inline src/auro-dropdown.js:59]: Nit-pick, please group the non-private and the private props separately for easy read/scanning.

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: .github/settings.yml (hunk 1 above)
Comment (jason-capsule42) [reply .github/settings.yml:6]: The whole readme needs to be done and will be handled outside of the repoUpgrade.

File: README.md (hunk 3 above)
Comment (jason-capsule42) [reply README.md:3]: Same comment as above.

File: README.md (hunk 4 above)
Comment (jason-capsule42) [reply README.md:72]: Same comment as above.

File: docs/api.md (hunk 5 above)
Comment (jason-capsule42) [reply docs/api.md]: These are not private intentionally. Hide() is necessary in cases where someone wants to programmatically close the dropdown from within the dropdown content. (e.g. dropdown menu - clicking a menu item selects the value but also closes the dropdown)

I am not entirely sure if show() should be exposed or not. Conversation for outside of the repoUpgrade though.

Comment (blackfalcon) [reply docs/api.md]: Can you please add descriptions to the JSDoc?

File: README.md (hunk 4 above)
Comment (blackfalcon) [reply README.md:72]: These are the things that get easily forgotten later as they will not be in another pull request and there is a lot of other things to review. 

If not addressed now, I'd like to at least see a new issue added to the repo about completing the documentation for the README.md file.

Comment (braven112) [reply README.md:72]: Heads up, I added a bullet point to the #8 to double check this and referenced this PR.

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**